jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/data/FeaturesConfigHelper.scala

```diff
@@ -1,23 +1,20 @@
 package ml4ir.inference.tensorflow.data
 
-import scala.collection.JavaConverters._
-import org.tensorflow.DataType
-
 case class NodeWithDefault(nodeName: String, defaultValue: String)
 
 object FeaturesConfigHelper {
-  implicit class ModelFeaturesConverter(modelFeatures: ModelFeatures) {
+  implicit class MFConverter(mf: ModelFeaturesConfig) {
     def toFeaturesConfig(tfRecordType: String): FeaturesConfig = {
+      val features = mf.features ++ (if (tfRecordType.equals("sequence")) List(mf.initialRank) else List.empty)
```

Comment:
Special handling for rank :/

Reply:
yes, it needs documentation explaining why